repository/filtering.go

```diff
@@ -33,15 +35,35 @@ func NewFiltersRepository(db *sql.DB) Filters {
 
 // Add adds new filter to database
 func (p *filtersRepo) Add(filter *Filter) error {
-	_, err := p.db.Exec(addQuery, filter.Name, filter.Type, filter.Configuration, filter.Reversed, filter.UserID)
+	filterID, err := uuid.NewUUID()
+	if err != nil {
+		return err
+	}
+	filter.ID = filterID
+

```

Comment:
To my mind it is unnecessary to create new variable filterID. 
What do you think about the usage of the following:
```go
filter.ID, err = uuid.NewUUID()
if err != nil {
	return err
}
```

Reply:
Does this example work?

Reply:
If we want do that in such way, we anyway need to declare var err before. So, not sure that is necessary change) WDYT?

Reply:
Sorry, You are right, I did not notice that you deleted _, err := p.db.Exec(addQuery, filter.Name, filter.Type, filter.Configuration, filter.Reversed, filter.UserID) and put it below.